src/main/menus/app.js

```diff
@@ -117,10 +124,13 @@ function createTemplate(config) {
     }, separatorItem, {
         label: 'Actual Size',
         role: 'resetZoom',
+        accelerator: 'CmdOrCtrl+0',
     }, {
         role: 'zoomIn',
+        accelerator: 'CmdOrCtrl+SHIFT+=',
```

Comment:
I'm not entirely convinced about this one, this is assuming all layouts are US or similar?
```suggestion
        accelerator: 'CmdOrCtrl+SHIFT++',
```
not sure if the above works or needs escaping somehow

Reply:
It's the same shortcut from the current desktop app, it reflects the fact that in order to get the `+` symbol, you would need to press `SHIFT+=`, unless you're using the numpad (in my case I actually don't have a numpad)

Reply:
I know, that happens in US, UK and French layout (among others), but German and Spanish layouts (and there might be others as well) don't work that way and might even be confusing (I can't press shift on a __shifted__ key)
![image](https://user-images.githubusercontent.com/1515906/109676031-9ae62c80-7b78-11eb-8e29-4f18c9032f66.png)
![image](https://user-images.githubusercontent.com/1515906/109676098-ac2f3900-7b78-11eb-989a-6cbfe6e4fab1.png)

Reply:
Fair point, I don't know if we can detect keyboard layout but we could do an override if so.
Otherwise we might need some UX input on this.

Reply:
@Willyfrog Can I merge as-is for now to fix the base issue? I think we have an outstanding ticket to investigate the keyboard shortcut.